### DISCUSSION TITLE
Warning: Repository Contains Malicious Code Leading to System Compromise

### DISCUSSION THREAD

**Discussion by mrahmadhassankhan** [community/community#135634] [Code Security]
### Select Topic Area

General

### Body

Hello GitHub Community,

I want to share a serious issue I encountered with a repository on GitHub that I believe contains malicious code. The repository in question is https://github.com/hacksider/Deep-Live-Cam.

Summary of the Issue:

    Action Taken: I cloned the repository and ran the code on my Ubuntu system after installing all dependencies.
    Malicious Activity: Later, I noticed that my microphone was activated without my consent, and a camera access prompt appeared on my screen. Additionally, critical system files were deleted, resulting in the loss of important data.
    Impact: This incident has led to significant disruption and data loss, with potential repercussions for my work and company.

Request for Assistance:

    Community Awareness: I want to warn others about the potential risks associated with this repository.

**Comment by ghostinhershell** [comment]
Hi @mrahmadhassankhan, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by mrahmadhassankhan** [reply]
Sure 🤝
